src/bio/bio_device.c

```diff
@@ -373,6 +366,16 @@ replace_dev(struct bio_xs_context *xs_ctxt, struct smd_dev_info *old_info,
 	D_ASSERT(owner_thread(bbs) != NULL);
 	spdk_thread_send_msg(owner_thread(bbs), setup_bio_bdev, old_dev);
 
+	/* Set the LED of the VMD device to OFF state */
+	rc = bio_led_manage(xs_ctxt, NULL, old_dev->bb_uuid, (unsigned int)CTL__LED_ACTION__SET,
+			    &led_state, 0);
```

Comment:
The intention here is to set old dev LED as off or set new dev LED as off? Please be aware that in most common case, the old dev has already been unplugged, and the new device might be plugged in the same PCIe slot.

Reply:
Thanks for that observation. So this isn't necessary as the old device will have been removed and the new one won't need it's LED state resetting. I will remove this. Done

Reply:
Actually, revisiting this. This is setting the new device LED to "OFF" ( as old_dev actually points to new_dev ) so this seems valid although now thinking about it I don't really see a use-case as the device will not be faulty and no point in resetting identify state.